taf/tools/cli/ocl.py

```diff
@@ -0,0 +1,14 @@
+import click
+import taf.tools.repo as repo_cli
+
+
+@click.group()
+def ocl():
```

Comment:
nitpick: this should be `olc` for Open Law Collections

Reply:
Yes, that was a typo. Done